test/containerized_test.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (C) 2022 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+import json
+import http.client as http_client
+
+import pytest
+
+from ovirt_imageio._internal import config
+from ovirt_imageio._internal import server
+from ovirt_imageio._internal.units import KiB
+
+from . import testutil
+from . import http
+
+
+TICKET_SIZE = 16 * KiB
+
+
+@pytest.fixture
+def tmp_ticket(tmpdir):
+    image = testutil.create_tempfile(tmpdir, name="disk.raw", size=TICKET_SIZE)
+    ticket = testutil.create_ticket(size=TICKET_SIZE, url=f'file://{image}')
+    ticket_path = tmpdir.join("file.json")
+    ticket_path.write(json.dumps(ticket))
+    return ticket["uuid"], str(ticket_path)
+
+
+@pytest.fixture
+def srv(tmp_ticket):
+    _, ticket_path = tmp_ticket
+    cfg = config.load("test/conf/daemon.conf")
+    s = server.Server(cfg, ticket=ticket_path)
+    s.start()
+    yield s
+    s.stop()
+
+
+@pytest.mark.parametrize("data", [
+    pytest.param(b"\0" * TICKET_SIZE, id="empty"),
+    pytest.param(
+        b"a" * (TICKET_SIZE // 2) + b"b" * (TICKET_SIZE // 2), id="full"),
+])
```

Comment:
We do not need it. I added it because I was checking the validity of the fixtures, and wanted to do more tests, so I parametrized it. And after that I decided to leave it.

But you are right in that it does not add any verification value. Let's simplify it again.